### ISSUE TITLE
Book showing impossible foods

### ISSUE THREAD

**Issue by willtarax** [Cazsius/Spice-of-Life-Carrot-Edition#111]
I am playing ATM6 to the Skies, and there are food items from Tinkers, such as the Jerky, which is impossible to make as Tinkers has not implemented their drying rack yet. This, of course, is wrecking havoc with our server chef who is trying to make it so everyone on the server has at least one of every type of food in the pack. These foods are not showing in JEI, nor in the creative tabs. I think the book should only show foods available to the players.

**Comment by juliand665** [comment]
Hm, I guess one option would be to have the mod scour through the creative tabs rather than all registered items when looking for food. Personally I would have expected Tinkers to not even register the jerky if it's not implemented yet lmao.

We could change the mod, but would blacklisting those foods be an okay solution for you?

**Comment by willtarax** [comment]
Yeah, that would be great! Thanks for your response! I would have replied sooner, but this response email fell into my spam folder >.<

**Comment by Karenpresci** [comment]
This is still happening to me in 2025, i tries to blacklist, but is impossible, because the game not read the name, its need the tag/prefab, so i cannot found it, because theres none of those items in my game.

![Image](https://github.com/user-attachments/assets/18fc0115-5722-492a-a7cb-62071e5111f2)

So, this is all the items impossible to craft/disabled that shows in my book:

Venison Stew Cup/Venison Stew
/Raw Venison Chops/Slice of Source Berry Pie/Source Berry Pie Ice Cream/Source Berry Gumme/Source Berry Cookie/Rock Candy/Slice of Passion Fruit Tart/Slice of Mulberry Pie/Honey Glazed Walnut/Jam Jar/Green Tea Leaves/Slice of Green Apple Pie/Glow Jam Jar/Slice of Gloomgourd Pie/Crab Rangoon/Cooked Venison Chops/Coconut Curry/Chunkwich/Slice of Chorus Pie/Chorus Muffin/Chopped Clover/Slice of Bleberry Pie.

You said the solution would be the mod only read the creative food items right? but how can i do it? thats only the blacklist config.

@Cazsius

**Comment by Mrbysco** [comment]
What did your blacklist config look like?

**Comment by Karenpresci** [comment]
> What did your blacklist config look like?

![Image](https://github.com/user-attachments/assets/0720e66b-8dd1-4841-93d8-70fb80e0aa27)
Simple like this, i tried put the name of the item show in the book, but i was already expecting not working, because the game use tags, not name, so probably those things has the tag like "modname:name_Raw_Venison_Chops".
Is that a way for me to make the game only look for food from Creative list like said before? its impossible to me know the tag/prefab name of something that doesnt exist :(( and its being very boring showing those impossible receipt

**Comment by Karenpresci** [comment]
So i just Found Out that using JEI, i can press F3+H to show Advanced Toolips, and then it show the item ID

![Image](https://github.com/user-attachments/assets/f923e77c-da57-4ec5-bf4a-d7f638a74732)

Its a "solution" for me to find every item that is impossible to craft, still, it will take a lot time... i still want a option to make the game only read my actual food items thought.
But its a solution for now

**Comment by Karenpresci** [comment]
![Image](https://github.com/user-attachments/assets/0daa100c-3588-4213-8ccb-e6b0a24112f2)

It will take a lot of time though.
